### PR TITLE
[AC-2568] Split billing history calls to separately call for invoices and transactions. Added paging buttons

### PR DESCRIPTION
**Pull request by cturnbull-bitwarden**
## 🎟️ Tracking

https://bitwarden.atlassian.net/browse/AC-2568

## 📔 Objective

Split billing history calls to separately call for invoices and transactions. Added paging buttons

## 📸 Screenshots

image
image

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

**Review by cyprain-okeke — COMMENTED** [review]
Looks Good

File: apps/web/src/app/billing/individual/billing-history-view.component.html [hunk 1]
```diff
@@ -2,17 +2,6 @@
   <h2 bitTypography="h2">
     {{ "billingHistory" | i18n }}
   </h2>
-  <button
```
Comment (amorask-bitwarden) [inline apps/web/src/app/billing/individual/billing-history-view.component.html:5]: 🎉 🎉 🎉

File: libs/common/src/billing/abstractions/account/account-billing-api.service.abstraction.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import {
+  BillingInvoiceResponse,
+  BillingTransactionResponse,
+} from "@bitwarden/common/billing/models/response/billing.response";
+
+export class AccountBillingApiServiceAbstraction {
```
Comment (amorask-bitwarden) [inline libs/common/src/billing/abstractions/account/account-billing-api.service.abstraction.ts:6]: 🥇 Into this, been wanting to split up that `billing-api.service`.

**Review by amorask-bitwarden — DISMISSED** [review]
Looks good to me, nice work.

**Review by cyprain-okeke — DISMISSED** [review]

**Review by amorask-bitwarden — APPROVED** [review]

**Pull request merged**